src/components/network/network-panel.js

```diff
@@ -135,8 +148,21 @@ genotet.NetworkPanel.prototype.setEdgeInfo_ = function(edge, container) {
     .text(edge.source.label);
```

Comment:
Yes, in renderer, each edge in edges_ has edge.source.label, edge.source/target are genotet.networkNode Objects. Maybe I should add some annotations for them.